main.py

```diff
@@ -688,7 +698,11 @@ async def noib(ctx):
     global _okib_channel
     global _okib_message_id
     
-    if ctx.message.author.roles[-1] <= _guild.get_role(params.PEON_ID):
+    #PUB OKIB
+    if ctx.channel ==  _bnet_channel:
+        pass
```

Comment:
I think this means anyone can !noib in the pub channel. That's not what we want, right?

Reply:
Not at all, it just bypass the peon or inferior check (which couldn't noib anyway) if we're in the pub channel.
THEN
Line 708 check still play his role => only gatherer, or shaman can !noib it in the first 2 hours.
Effectively after 2 hours anyone can !noib it. Do we still want to limit that type of !noib to a certain role or @everyone is fine with that ?

Reply:
oooh you're right, mb. Ok, I think maybe limit !noib to pub hosts or above just to keep it consistent with the !okib permissions, like on line 625

Reply:
Alr, added that in line 702 if it's not satisfied it'll end up with regular peon check and reject the non pub host or above fool

Reply:
lol, funny way to do it but sounds good